Specification/Definitions/types.md

```diff
@@ -14,7 +14,7 @@ Instruction and ref specification use the following common types.
 | **Aliquot**                 | `"growth_plate/A1"`     | an Autoprotocol container and a well index delimited with a `/` represented as a String
 | **Container**               | `"growth_plate"`        | an Autoprotocol container referenced in the refs section of the protocol represented as a String
 | **Quantity**  e.g. `Volume` | `"5:microliters"`       | a magnitude and a unit delimited with a `:` represented as a String
-| **Compound**                | `"InChI=1S/CH4/h1H4"`   | a chemical compound definition, following the IUPAC Standard InChI format.
+| **Compound**  e.g. `InChI`  | `"InChI=1S/CH4/h1H4"`   | a chemical compound definition, following the IUPAC Standard InChI format.
```

Comment:
hmm this is technically not right, we should probably change the example value to reflect the ASC

Reply:
Oh. You mean with both 'format' and 'value'?

Reply:
Yup! Its more awkward and verbose but yea... Unless we do some automatic inference? 
In any case, yea let's fix this to reflect the asc